app/i18n/docs/en/InfoModals/TicketAutoBuyer.md

```diff
@@ -1,6 +1,6 @@
 # Automatic Purchase Information
 
-**Balance to Maintain** If your balance is lower than this number, you will not buy tickets. The default of 0 will use all the funds in your account to buy tickets
+**Balance to Maintain** If your balance is lower than this number, you will not buy tickets. The default of 0 will use all the funds in your account to buy tickets.
```

Comment:
This should be in a dedicated PR, since it's a different change than the rest of this PR.